requirements/agent.txt

```diff
@@ -0,0 +1 @@
+elastic-agent-client@git+https://github.com/elastic/python-elastic-agent-client@main
```

Comment:
For now we just pick `main`, later we'll have more solid plan, hopefully

Reply:
"Always releasable" enforced here 🤙